java/src/main/java/ai/rapids/cudf/PinnedMemoryPool.java

```diff
@@ -224,7 +224,7 @@ private PinnedMemoryPool(long poolSize, int gpuId, boolean setCuioHostMemoryReso
     }
     this.poolHandle = Rmm.newPinnedPoolMemoryResource(poolSize, poolSize);
     if (setCuioHostMemoryResource) {
-      Rmm.setCuioPinnedPoolMemoryResource(this.poolHandle);
+      Rmm.setCudfPinnedPoolMemoryResource(this.poolHandle);
```

Comment:
we should also change the variable name `setCuioHostMemoryResource` -> `setCudfPInnedPoolMemoryResource`. This won't break the spark-rapids build, as it will still find the right method, but at least it will show up in IDEs and what not.

Thanks for making java changes!